testimony/__main__.py

```diff
@@ -33,13 +33,19 @@ def parse_args():
         '-j', '--json', action='store_true', help='JSON output')
     parser.add_argument(
         '-n', '--nocolor', action='store_true', help='Do not use color option')
+    parser.add_argument(
+        '-t',
+        '--tags',
+        help=('comma separated tags to search.  Note: Always run this '
```

Comment:
This context is already inside parentheses, and there is no need to wrap the string on parentheses again. (Just a curiosity)